client/components/Editor/index.jsx

```diff
@@ -57,11 +62,12 @@ class Editor extends Component {
             case 'slides':
                 return (
                     <Slides
-                        scenarioId={this.props.match.params.id}
-                        className="active"
+                        scenarioId={scenarioId}
                         updateEditorMessage={this.updateEditorMessage}
                     />
                 );
+            case 'preview':
+                return <Scenario scenarioId={scenarioId} />;
```

Comment:
Nice

Reply:
:D looking at this again, I'm realizing I'm missing some good object shorthand opportunities...

Reply:
Oh never mind, what I was thinking isn't a thing in JSX.